### PR TITLE
Add community files and fix Viewport

### PR DESCRIPTION
**Pull request by drac**
## Proposed Changes

-  add community files
-  add issue and pull request templates
-  update npm packages
-  use official threejs OrbitControls on Viewport
-  use HemisphereLight instead PointLight

### CODE REVIEW

File: src/utils/addLine.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
- * @type {Function} Draws line on the scene.
- *
- * @param {Object} scene
- * @param {Array} coor1
- * @param {Array} coor2
- * @param {Number} lineColor
- *
- */
-
-const addLine = (scene, coor1, coor2, lineColor) => {
```
Comment (kbsezginel) [inline src/utils/addLine.js]: We might need a similar function in the future to show bonds between atoms but I think it's fine to remove this now.

File: package.json [hunk 2]
```diff
@@ -44,11 +44,12 @@
...
-    "eslint-plugin-jest": "^22.7.2",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.14.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react": "^7.14.3",
+    "glob-gitignore": "^1.0.14",
     "prettier": "^1.18.2"
   }
```
Comment (hamzaerbay) [inline package.json]: What do you think for the pre-commit check? 
https://prettier.io/docs/en/precommit.html#option-2-pretty-quick-https-githubcom-azz-pretty-quick

Comment (drac) [reply package.json]: I think it's a good idea. Can you implement it?

Comment (hamzaerbay) [reply package.json]: Sure, let’s merge your pr then I will create a PR for that.

**Pull request merged**